widget/embedded/src/constants/searchParams.ts

```diff
@@ -5,6 +5,7 @@ export enum SearchParams {
   TO_TOKEN = 'toToken',
   FROM_AMOUNT = 'fromAmount',
   AUTO_CONNECT = 'autoConnect',
+  BLOCKCHAIN = 'blockchain',
```

Comment:
Add a comment that this is for custom tokens.

Reply:
I added a comment